src/main/java/org/opensearch/security/auditlog/sink/WebhookSink.java

```diff
@@ -269,14 +275,13 @@ protected boolean doPost(String url, String payload) {
 
 		HttpPost postRequest = new HttpPost(url);
 
-		StringEntity input = new StringEntity(payload, StandardCharsets.UTF_8);
-		input.setContentType(webhookFormat.contentType.toString());
+		StringEntity input = new StringEntity(payload, webhookFormat.contentType);
```

Comment:
Not sure how `StandardCharsets.UTF_8` is important, but we could go with `webhookFormat.contentType.withCharset(StandardCharsets.UTF_8)`

Reply:
Thanks for catching this, there's no constructor in the new StringEntity that will take all 3 and there's no `setContentType` in v5